git-merge-conflict.md

```diff
@@ -0,0 +1,52 @@
+# Merge Conflict Carnival
+## Overview
+This year the Ada staff has generously agreed to contribute their culinary skills to a coding bootcamp booth offering food & info pamphlets at the [Georgetown Carnival](http://www.georgetowncarnival.com/). It's the day before the festivities kick off and _someone's_ Cavalier King Charles Spaniel has done his best to shred all of their recipes! Oh no! Can you help Ada staff put everything back together and save Carnival?
+
+In this activity you'll be working with your fellow Adies to put together multiple scraps from a single recipe using Git. You'll accomplish this by successively merging in content from each individuals' copy of the recipes repo. Once all the of merges are finished (and any merge conflicts resolved) you'll have complete recipe document which can be submitted to the original repository with a pull request.
+
+## Instructions
+### Baseline Setup
+Before we can start putting the recipe together we need to make sure that everyone in your group is working from the same starting point and can effectively contribute to the recipe building effort.
+
+1. **Split into groups**: Each table is marked with a particular group number and everyone at that table will be part of the group. There should be six people in each group.
+1. **Fork the activity repo**: One person in your group should fork the [carnival-recipes](https://github.com/Ada-C7/carnival-recipes) repo and then add everyone else as a collaborator.
+1. **Clone the forked repo**: Everyone else in your group should clone the forked repository.
+1. **Create the recipe file**: To keep things better organized, one person in your group should take the [title scrap](https://github.com/Ada-C7/carnival-recipes/#recipe-scraps-for-each-group) and copy its contents into another file in the `recipes/` folder. This new file should be named appropriately based on the recipe title. **You should also delete the title scrap once its contents have been moved into the recipe file.**
+1. **Pull the recipe file**: Once the recipe file has been created and committed, everyone else in your group should `git pull` that commit to their local copy of the repository.
```

Comment:
I had also been thinking that we could just provide these files in the repo for them, it might save on some potential confusion.